x-pack/plugins/security_solution/common/detection_engine/rule_management/api/rules/coverage_overview/request_schema.ts

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import * as t from 'io-ts';
+import { NonEmptyString } from '@kbn/securitysolution-io-ts-types';
+
+export type CoverageOverviewRuleActivity = t.TypeOf<typeof CoverageOverviewRuleActivity>;
+export const CoverageOverviewRuleActivity = t.union([
+  t.literal('enabled'),
+  t.literal('disabled'),
+  t.literal('available'),
+]);
+
+export type CoverageOverviewRuleSource = t.TypeOf<typeof CoverageOverviewRuleSource>;
+export const CoverageOverviewRuleSource = t.union([
+  t.literal('prebuilt'),
+  t.literal('custom'),
+  t.literal('customized'), // Customized prebuilt rule
+]);
```

Comment:
Should these be enums instead? Could be a bit better DX if you want to handle individual values.
There's an `enumeration` function that can generate an io-ts schema from an enum.

Reply:
Good suggestion 👍 . I'll convert them to enums.